cognite_toolkit/_cdf_tk/utils.py

```diff
@@ -772,6 +774,56 @@ def verify_security_categories(
                 f"Security category {e} does not exist. You need to create it first.", e.args[0]
             ) from e
 
+    @overload
```

Comment:
This is part of the prototype, and should ideally in the `prototypes` modules. But it is not used anywhere and thus not exposed, so bugs here will not influence the existing stable code.